lib/beaker/hypervisor/vagrant.rb

```diff
@@ -48,7 +48,7 @@ def make_vfile hosts, options = {}
         v_file << "    v.vm.box_download_insecure = '#{host['box_download_insecure']}'\n" unless host['box_download_insecure'].nil?
         v_file << "    v.vm.box_check_update = '#{host['box_check_update'] ||= 'true'}'\n"
         v_file << "    v.vm.synced_folder '.', '/vagrant', disabled: true\n" if host['synced_folder'] == 'disabled'
-        v_file << private_network_generator(host)
+        v_file << private_network_generator(host) unless (host['private_network_disable'].nil? and host['private_network_disable'] == 'true' )
 
```

Comment:
should the check here to add the private_network_generator to the v_file just look for whether host['private_network_disable'] is set to true?

Reply:
@mchllweeks I definitely need to add some test. :+1: